ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaPlayerTokenPlayerActivity.java

```diff
@@ -40,33 +45,44 @@
  */
 public class OoyalaPlayerTokenPlayerActivity extends Activity implements Observer, EmbedTokenGenerator {
   final String TAG = this.getClass().toString();
-
+  private static final int PERMISSIONS_REQUEST_WRITE_EXTERNAL_STORAGE = 1;
   String EMBED = null;
   String PCODE = null;
   String DOMAIN = null;
-  private final String ACCOUNT_ID = "accountID";
+  Boolean AUTOPLAY = false;
+  private String ACCOUNT_ID = "";
 
   /*
    * The API Key and Secret should not be saved inside your applciation (even in git!).
    * However, for debugging you can use them to locally generate Ooyala Player Tokens.
    */
-  private final String APIKEY = "Use this for testing, don't keep your secret in the application";
-  private final String SECRET = "Use this for testing, don't keep your secret in the application";
+  private String APIKEY = "BjcWYyOu1KK2DiKOkF41Z2k0X57l.0I-V4";
```

Comment:
remove api key and secret values, these are supposed to be secret. Are this from a test account? If not you'll need to completely remove them from GIT.